### DISCUSSION TITLE
an Android APP can connect using Psiphone but not through StrongSwan

### DISCUSSION THREAD

**Discussion by farhadsakhaei** [strongswan/strongswan#312] [General]
Hi,
I installed Waze app on Android, it seems Waze is banned by filtering in my country
I can use Waze when I connect through Psiphon VPN but I can't use it through my Strongswan app
Everything is ok on my Strongswan and it works for other applications like Youtube and Telegram,
But for Waze it doesn't work
Any idea ?
Thank you

**Comment by Thermi** [comment]
No, no idea why that is. You need to disect the problem at the different network layers. Good effort.
